_docs/automation-time-apis.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Time Trigger APIs
+subtitle: The API specifications for Time Triggers
+author: ryanh
+tags: [api, time, triggers]
+---
+
+These APIs allow you to schedule an action to occur at a specific minute, and to cancel the task. All the APIs in this section are prefixed by `Automation-time`. 
+
+
+## Launched
+
+### Cancel a Task
```

Comment:
i'd consider re-jigging this by importance over alphabetical

Reply:
I was thinking more that the list will continue to grow. So cancel should either be on top or bottom. I am guessing yhis means you think on bottom :)